src/actions/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import Button from '@material-ui/core/Button';
+import { Redirect } from 'react-router-dom'
+import userLoginFetch from './actions.js'
+import userLogin from './actions.js'
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { 
+      authenticated: null,
+      username: "",
+      password: "",
+    };
+    // this.checkAuthentication = this.checkAuthentication.bind(this);
+    this.login = this.login.bind(this);
+  }
+
+  handleChange = event => {
+    this.setState({[event.target.name]: event.target.value});
```

Comment:
We're using the other file right? @Tonewall

Reply:
Yeah and keep the login.js

Reply:
Yes. Fixed in commit.